benchmark/benchmarks.jl

```diff
@@ -40,51 +51,58 @@ for subdir ∈ subdirs
       push!(names_sqd, name)
       println(name)
 
-      p_amd      = amd(A)
-      p_symamd   = symamd(A)
-      p_metis, _ = Metis.permutation(A)
-      p_symrcm   = symrcm(A)
-      p_classic  = collect(1:size(A,1))
-
-      ldlt_amd     = ldl(A, p_amd)
-      ldlt_symamd  = ldl(A, p_symamd)
-      ldlt_metis   = ldl(A, p_metis)
-      ldlt_symrcm  = ldl(A, p_symrcm)
-      ldlt_classic = ldl(A, p_classic)
-
-      push!(nnz_sqd, nnz_A)
-      push!(ratio_amd, nnz(ldlt_amd) / nnz_A)
-      push!(ratio_symamd, nnz(ldlt_symamd) / nnz_A)
-      push!(ratio_metis, nnz(ldlt_metis) / nnz_A)
-      push!(ratio_symrcm, nnz(ldlt_symrcm) / nnz_A)
-      push!(ratio_classic, nnz(ldlt_classic) / nnz_A)
+      # p_amd      = amd(A)
+      # p_symamd   = symamd(A)
+      # p_metis, _ = Metis.permutation(A)
+      # p_symrcm   = symrcm(A)
+      # p_classic  = collect(1:size(A,1))
+
+      # ldlt_amd     = ldl(A, p_amd)
+      # ldlt_symamd  = ldl(A, p_symamd)
+      # ldlt_metis   = ldl(A, p_metis)
+      # ldlt_symrcm  = ldl(A, p_symrcm)
+      # ldlt_classic = ldl(A, p_classic)
+
+      # push!(nnz_sqd, nnz_A)
+      # push!(ratio_amd, nnz(ldlt_amd) / nnz_A)
+      # push!(ratio_symamd, nnz(ldlt_symamd) / nnz_A)
+      # push!(ratio_metis, nnz(ldlt_metis) / nnz_A)
+      # push!(ratio_symrcm, nnz(ldlt_symrcm) / nnz_A)
+      # push!(ratio_classic, nnz(ldlt_classic) / nnz_A)
+
+      SUITE["no_ordering"][name] = @benchmarkable collect(1:size($A,1))
+      SUITE["amd"][name] =  @benchmarkable amd($A)
```

Comment:
Here it's the same but with the ordering given by `amd(A)`. See the code that you commented above.